cli/src/cmd/utils.rs

```diff
@@ -228,3 +230,33 @@ pub fn has_batch_support(chain: u64) -> bool {
     }
     true
 }
+
+pub trait LoadConfig {
```

Comment:
I like this!

the function names make sense to me. an alternative that comes to mind is `load_config_verbose()`, but I don't think we'll ever log anything other than warning so lets stick with that here